apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam.env.$envParam.queues/route.tsx

```
@@ -184,11 +185,21 @@ export const action = async ({ request, params }: ActionFunctionArgs) => {
   switch (action) {
     case "environment-pause":
       const pauseService = new PauseEnvironmentService();
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)